drivers/spi/spi_mcux_flexcomm.c

```diff
@@ -83,6 +92,10 @@ static void spi_mcux_transfer_next_packet(const struct device *dev)
 	spi_transfer_t transfer;
 	status_t status;
 
+	#ifdef CONFIG_PM_POLICY_DEVICE_CONSTRAINTS
+	pm_policy_device_power_lock_get(dev);
+	#endif
```

Comment:
I don't understand why the #ifdef is needed at all? the header file already defined empty versions of the function if CONFIG_PM is off.